### ISSUE TITLE
Descriptions of each box in the blueprints

### ISSUE THREAD

**Issue by noa-codes** [ai-infrastructure-alliance/blueprints#10]
It would be great to have descriptions of what each box in the blueprints represents, with examples of tools. I haven't heard many of these terms being used in the MLOps community.

For example, how should I think about the "Data Engineering Engine" vs. the "Data Engineering Orchestration Pipeline"? If the "orchestrator" is something like dbt + Airflow, what is the "engine"? The database itself?

**Comment by mariyadavydova** [comment]
@noa-codes Absolutely! One of our working groups is currently working on refining the definitions and updating the blueprint itself.
